armadillo/src/main/java/org/molgenis/armadillo/ArmadilloServiceApplication.java

```diff
@@ -3,12 +3,28 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.retry.annotation.EnableRetry;
+import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.util.UrlPathHelper;
 
 @SpringBootApplication(scanBasePackages = "org.molgenis")
 @EnableRetry
-public class ArmadilloServiceApplication {
+public class ArmadilloServiceApplication implements WebMvcConfigurer {
```

Comment:
Don't know if it matters, but you could also use https://github.com/molgenis/molgenis-service-armadillo/blob/master/armadillo/src/main/java/org/molgenis/armadillo/interceptor/WebConfig.java which is also a WebMvcConfigurer?

Generally, this is why I don't like Spring. To much indirection.

Reply:
Good point, forgot to look for another implementation